### DISCUSSION TITLE
TB Send later extension - does not send messages by default / PGP Encrypt

### DISCUSSION THREAD

**Discussion by Seriouslaughbore** [Extended-Thunder/send-later#332] [Q&A]
Hi, I have an issue with Send later on TB:

I can only send messages when I click send immediately, otherwise it hangs in the drafts forever. 

I do have other extensions installed - Cardbook/Envoyer plus tard/Filelink provider for Dropbox/Quicktext/Shrunked Image resizer/Signature Switch/TBSync/Textmaker/XNote++  Also use encryption -- OpenPGP.

Would any of these features interfere with "Envoyer plus tard"/Send later?

Here's a copy of the error console:

OpenGL compositor Initialized Succesfully.
Version: 2.1 INTEL-14.7.13
Vendor: Intel Inc.
Renderer: Intel Iris Pro OpenGL Engine
FBO Texture Target: TEXTURE_2D 2
NS_ERROR_FAILURE: Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIMessenger.msgHdrFromURI] SearchIntegration.jsm:537
TypeError: hdr is null background.js:1270:21
WARN [SendLater]: Unable to find message encryptionincompattitle in locale fr 86 static.js:51:13
TypeError: gDBView is null
4 xnote-columnnote.js:82:7
DOMException: IDBDatabase.transaction: Can't start a transaction on a closed database tabmail.js:1110:21
OverrideError: An entry font-size-label of type message is already defined in this bundle
OverrideError: An entry window-close-key of type message is already defined in this bundle
OverrideError: An entry startup-label of type message is already defined in this bundle
OverrideError: An entry focus-search-shortcut of type message is already defined in this bundle
OverrideError: An entry close-button of type message is already defined in this bundle
OverrideError: An entry font-size-label of type message is already defined in this bundle
OverrideError: An entry window-close-key of type message is already defined in this bundle
OverrideError: An entry startup-label of type message is already defined in this bundle
OverrideError: An entry focus-search-shortcut of type message is already defined in this bundle
OverrideError: An entry close-button of type message is already defined in this bundle
OverrideError: An entry font-size-label of type message is already defined in this bundle
OverrideError: An entry window-close-key of type message is already defined in this bundle
OverrideError: An entry startup-label of type message is already defined in this bundle
OverrideError: An entry focus-search-shortcut of type message is already defined in this bundle
OverrideError: An entry close-button of type message is already defined in this bundle
NotSupportedError: CustomElementRegistry.define: 'conversation-browser' has already been defined as a custom element conversation-browser.js:853
DOMException: IDBDatabase.transaction: Can't start a transaction on a closed database tabmail.js:1110:21
Une chaîne vide a été transmise à « getElementById() ». browser-custom-element.js:411:22
SyntaxError: redeclaration of const THUNDERBIRD_THEME_PREVIEWS
2 aboutAddonsExtra.js:1:1
TypeError: prin.URI is null
ConduitsParent.jsm:123:58
Error: Unknown sender or wrong actor for recvCreateProxyContext ConduitsParent.jsm:321:13
2147942487
OpenGL compositor Initialized Succesfully.
Version: 2.1 INTEL-14.7.13
Vendor: Intel Inc.
Renderer: Intel Iris Pro OpenGL Engine
FBO Texture Target: TEXTURE_2D

Thank you for helping out!

**Comment by Seriouslaughbore** [comment]
It seems another thread discusses the issue on https://github.com/Extended-Thunder/send-later/discussions/200 I'm sorry for the duplication. 
Having fluid interoperability of encryption and send later features would be amazing! 

A bugzilla report (https://bugzilla.mozilla.org/show_bug.cgi?id=1681168) which has been filed by @farfallosa

A temporary fix to disable OpenPGP when using SendLater has been posted. However I don't know how to install it.

**Comment by jikamens** [comment]
This is currently basically impossible to implement because none of the relevant functionality is exposed to extensions in APIs. I don't know if/when this will change. It may never change, so I'm going to close this discussion.
